package/features/jupyter.feature

```diff
@@ -27,15 +27,15 @@
 # limitations under the License.
 
 
-Feature: Running viz in Jupyter notebook
+# Feature: Running viz in Jupyter notebook
```

Comment:
I've commented on this behave test for now, because it is not working and I'm working on it in a separate branch.